docusaurus/docs/dev-docs/api/rest/guides/populate-creator-fields.md

```diff
@@ -0,0 +1,69 @@
+---
+title: How to populate creator fields
+description: Learn how to populate creator fields such as createdBy and updatedBy by creating a custom controller that leverages the populate parameter.
+---
+
+# 🛠️ How to populate creator fields such as `createdBy` and `updatedBy`
+
+The creator fields `createdBy` and `updatedBy` are removed from the [REST API](/dev-docs/api/rest) response by default. These 2 fields can be returned in the REST API by activating the `populateCreatorFields` parameter at the content-type level.
+
+:::note
+
+The `populateCreatorFields` property is not available to the GraphQL API.
+:::
+
+To add `createdBy` and `updatedBy` to the API response:
+
+1. Open the content-type `schema.json` file.
+2. Add `"populateCreatorFields": true` to the `options` object:
+
+  ```json
+  "options": {
+      "draftAndPublish": true,
+      "populateCreatorFields": true
+    },
+  ```
+
+3. Save the `schema.json`.
+4. Open the controller `[collection-name].js` file inside the corresponding API request.
+5. Add the following piece of code, and make sure you replace the `[collection-name].js` with proper collection name:
+
+  ```js
```

Comment:
Will work with you on creating a more "best practice" guide here as there are quite a few concerning things in this one.